src/libs/ReportUtils.ts

```diff
@@ -7817,8 +7817,8 @@ function hasMissingInvoiceBankAccount(iouReportID: string): boolean {
     return invoiceReport?.ownerAccountID === currentUserAccountID && isEmptyObject(getPolicy(invoiceReport?.policyID)?.invoice?.bankAccount ?? {}) && isSettled(iouReportID);
 }
 
-function isExpenseReportManagerWithoutParentAccess(report: OnyxEntry<Report>) {
-    return isExpenseReport(report) && report?.hasParentAccess === false && report?.managerID === currentUserAccountID;
+function isExpenseReportWithoutParentAccess(report: OnyxEntry<Report>) {
+    return isExpenseReport(report) && report?.hasParentAccess === false;
```

Comment:
This check function and its usage led to the following issue:
- https://github.com/Expensify/App/issues/52396

which we fixed by removing the `isExpenseReportWithoutParentAccess` check after this [discussion](https://github.com/Expensify/App/issues/52396#issuecomment-2475313562).